src/chapters/exceptions/exceptions-control-flow.rst

```diff
@@ -0,0 +1,72 @@
+Exceptions-Control-Flow
+=======================
+
+In some instances we receive input from a user, and we can't always ensure they will type in something that will run perfectly with our code.
+
+JavaScript gives us some control flow tools to handle exceptions, by catching thrown exceptions.
+
+When we are anticipating an exception, and we want our program to continue running, and not experience the runtime error we can tell JavaScript to **try** to run a block of code, and if a specific exception is thrown to **catch** the exception and to run a different block of code.
```

Comment:
This sentence is confusing.

"We may be anticipating an exception, but still want our program to continue running and not experience a runtime error. We can tell JavaScript to **try** to run a block of code and if an exception is thrown, to **catch** the exception and run a different block of code."

Reply:
fixed